Author: Yun Peng <pcloudy@google.com>
Date:   Fri Sep 25 10:34:13 2020 +0200

    Add zlib.BUILD

diff --git a/zlib.BUILD b/zlib.BUILD
new file mode 100644
index 0000000000..95ae7c5a89
--- /dev/null
+++ b/zlib.BUILD
@@ -0,0 +1,36 @@
+cc_library(
+    name = "zlib",
+    srcs = [
+        "adler32.c",
+        "compress.c",
+        "crc32.c",
+        "deflate.c",
+        "infback.c",
+        "inffast.c",
+        "inflate.c",
+        "inftrees.c",
+        "trees.c",
+        "uncompr.c",
+        "zutil.c",
+    ],
+    hdrs = [
+        "crc32.h",
+        "deflate.h",
+        "gzguts.h",
+        "inffast.h",
+        "inffixed.h",
+        "inflate.h",
+        "inftrees.h",
+        "trees.h",
+        "zconf.h",
+        "zlib.h",
+        "zutil.h",
+    ],
+    includes = [
+        ".",
+    ],
+    linkstatic = 1,
+    visibility = [
+        "//visibility:public",
+    ],
+)
